spec.emu

```diff
@@ -114,7 +114,7 @@ contributors: Google, Ecma International
           </tr>
           <tr>
             <td>*"japanese"*</td>
-            <td>Japanese Imperial calendar, era system hybridised with *"gregory"*. Month numbers, month codes, and days are the same as in the ISO 8601 calendar, extended proleptically before their introduction in ISO year 1873. Imperial era names only extend as far back as the Meiji period (starting in ISO year 1868) during which calendar reforms took place. The arithmetic year, and the years and eras before ISO year 1868, are identical to *"gregory"*.</td>
+            <td>Japanese Imperial calendar, era system hybridised with *"gregory"*. Month numbers, month codes, and days are the same as in the ISO 8601 calendar, extended proleptically before their introduction in ISO year 1873. Imperial era names only extend as far back as the Meiji period (starting in ISO year 1868) during which calendar reforms took place. All era names are defined by the Japanese government. The arithmetic year, and the years and eras before ISO year 1868, are identical to *"gregory"*.</td>
```

Comment:
Maybe "Modern era names are defined..."? (The decision to use era names only since 1868 is JS's concession to practical implementation concerns, and `bce`/`ce` are not defined by the Japanese government)